server/index.js

```diff
@@ -2,20 +2,31 @@ const express = require('express');
 const app = express();
 const port = 8080;
 const queryAPI = require('./atelier.js');
+const uploadPhotoToCloudinary = require('./cloudinary.js');
 app.use(express.static('public'));
-app.use(express.json());
+app.use(express.json({limit: '50mb'}));
+app.use(express.urlencoded({limit: '50mb'}));
```

Comment:
@jabrake @matthewwrobel @alizehrehman I've recently modified these two lines of code so that the server can handle a larger payload of data from AJAX post requests.